src/Metadata/TargetsMetadata.php

```diff
@@ -116,6 +116,13 @@ protected static function getSignedCollectionOptions(): array
                                     new NotBlank(),
                                 ]),
                             ]),
+                            'path_hash_prefixes' => new Optional([
```

Comment:
So they are not mutually exclusive here?

Reply:
They are. But this information is never _used_ by TargetsMetadata, so it make more sense to do the validation in DelegatedRole, where it's actually meaningful.